src/main/resources/init_db.sql

```diff
@@ -18,3 +18,25 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+DROP TABLE IF EXISTS `cars`;
+CREATE TABLE `cars` (
+                        `id` bigint NOT NULL AUTO_INCREMENT,
+                        `model` varchar(150) DEFAULT NULL,
+                        `manufacturer_id` bigint DEFAULT NULL,
+                        `is_deleted` tinyint NOT NULL DEFAULT '0',
+                        PRIMARY KEY (`id`),
+                        KEY `cars_manufacturers_fk` (`manufacturer_id`),
+                        CONSTRAINT `cars_manufacturers_fk` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8mb3;
+
+DROP TABLE IF EXISTS `cars_drivers`;
+CREATE TABLE `cars_drivers` (
+                                `car_id` bigint NOT NULL,
+                                `driver_id` bigint NOT NULL,
+                                KEY `cars_drivers_cars_fk` (`car_id`),
+                                KEY `cars_drivers_drivers_fk` (`driver_id`),
+                                CONSTRAINT `cars_drivers_cars_fk` FOREIGN KEY (`car_id`) REFERENCES `cars` (`id`),
+                                CONSTRAINT `cars_drivers_drivers_fk` FOREIGN KEY (`driver_id`) REFERENCES `drivers` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3;
+
```

Comment:
```suggestion
```
1) don't need write: ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8mb3
2) use upper case for typ of fields

Reply:
Do you mean I should delete line 31 and line 41 ?